### PR TITLE
Content edits in the Angular 2 Guide.

### PR DESCRIPTION
**Pull request by cfranger**
@juleskremer Here are my edits to the "Forms" page.

### CODE REVIEW

File: public/docs/ts/latest/guide/forms.jade [hunk 1]
```diff
@@ -1,83 +1,90 @@
...
 :marked
-  We’ve all used a form to log in, submit a help request, place an order, book a flight,
-  schedule a meeting, and perform countless other data entry tasks.
   Forms are the mainstay of business applications.
+  You use forms to log in, submit a help request, place an order, book a flight,
+  schedule a meeting, and perform countless other data-entry tasks.
 
-  Any seasoned web developer can slap together an HTML form with all the right tags.
-  It's more challenging to create a cohesive data entry experience that guides the
+  Any seasoned web developer can create an HTML form with all the right tags.
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/forms.jade]: Can we change the tone here to remove the "seasoned web developer" -> It is important to create a data entry experience that guides...

File: public/docs/ts/latest/guide/forms.jade [hunk 2]
```diff
@@ -1,83 +1,90 @@
...
+  It's more challenging to create a cohesive data-entry experience that guides the
   user efficiently and effectively through the workflow behind the form.
 
-  *That* takes design skills that are, to be frank, well out of scope for this guide.
+  *That* takes design skills that are, to be frank, well out of scope for this page.
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/forms.jade]: Can we change the tone here:  "Designing forms requires x skills which are out of scope for this page"

File: public/docs/ts/latest/guide/forms.jade [hunk 3]
```diff
@@ -1,83 +1,90 @@
...
-
-  Run the <live-example></live-example>.
+  *two-way data binding, change tracking, validation, and error handling*,
+  which is covered in this page on Angular forms.
+  <!-- CF: For the last two paragraphs, would the following rewrite work:
+   "Creating a cohesive data-entry experience requires design skills (which are out of
+   scope for this page), as well as framework support for
+   *two-way data binding, change tracking, validation, and error handling*,
+   which you'll learn about on this page."
+   -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/forms.jade]: yes please

File: public/docs/ts/latest/guide/forms.jade [hunk 4]
```diff
@@ -1,83 +1,90 @@
...
-  Many of us will build forms by writing templates in the Angular [template syntax](./template-syntax.html) with
-  the form-specific directives and techniques described in this guide.
+  You can build forms by writing templates in the Angular [template syntax](./template-syntax.html) with
+  the form-specific directives and techniques described in this page.
 
 .l-sub-section
   :marked
-    That's not the only way to create a form but it's the way we'll cover in this guide.
+    That's not the only way to create a form but it's the way that's covered in this page.
+//    CF: Is this note necessary?
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/forms.jade]: It is but it should be more descriptive that there two choices: template-driven and reactive.

**Comment by cfranger** [comment]
@juleskremer thanks for your comments. I committed some updates based on your feedback. I have a few more questions in the page--please search for "CF:"

**Comment by cfranger** [comment]
@juleskremer My latest commit is an edit of the Glossary page. I added several comments with questions. Search for "CF:". Thanks!

File: public/docs/ts/latest/glossary.jade [hunk 5]
```diff
@@ -150,10 +151,11 @@ a#aot
     The *component* is one of the most important building blocks in the Angular system.
     It is, in fact, an Angular [directive](#directive) with a companion [template](#template).
 
-    You apply the `!{_at}Component` !{_decoratorLink} to
+    Apply the `!{_at}Component` !{_decoratorLink} to
     the component class, thereby attaching to the class the essential component metadata
     that Angular needs to create a component instance and render it with its template
     as a view.
+    <!-- CF: What do "it" and "its" refer to? The component class, component metadata, or component instance? -->
```
Comment (juleskremer) [inline public/docs/ts/latest/glossary.jade]: the component with it's template.

File: public/docs/ts/latest/glossary.jade [hunk 6]
```diff
@@ -164,7 +166,10 @@ a#aot
 .l-sub-section
   :marked
     The practice of writing compound words or phrases such that each word is separated by a dash or hyphen (`-`).
-    This form is also known as [kebab-case](#kebab-case).
+    This form is also known as kebab-case.
+    <!-- CF: I removed the link to the kebab-case glossary entry because the entry has no additional
```
Comment (juleskremer) [inline public/docs/ts/latest/glossary.jade]: lgtm

File: public/docs/ts/latest/glossary.jade [hunk 7]
```diff
@@ -241,23 +250,26 @@ a#aot
...
     for us and handle all the dependencies.
-    If "A" needs "B" and "B" needs "C", the system resolves that chain of dependencies
-    and returns a fully prepared instance of "A".
+    If "A" needs "B" and "B" needs "C," the system resolves that chain of dependencies
+    and returns a fully prepared instance of "A."
 
     Angular provides and relies upon its own sophisticated
-    [dependency injection](!{docsLatest}/guide/dependency-injection.html) system
+    dependency-injection system
+    <!-- CF: I removed the link here because the same link is at the end of this definition. -->
```
Comment (juleskremer) [inline public/docs/ts/latest/glossary.jade]: lgtm

File: public/docs/ts/latest/glossary.jade [hunk 8]
```diff
@@ -618,22 +634,21 @@ a#snake-case
...
 .l-sub-section
   :marked
-    For data or logic that is not associated
-    with a specific view or that you want to share across components, build services.
-
-    Applications often require services such as a hero data service or a logging service.
-
-    A service is a class with a focused purpose.
-    We often create a service to implement features that are
+    <!-- CF: Removed the first sentence/paragraph because it's redundant.
```
Comment (juleskremer) [inline public/docs/ts/latest/glossary.jade]: lgtm

File: public/docs/ts/latest/glossary.jade [hunk 9]
```diff
@@ -738,20 +754,22 @@ a#Y
...
-      Zones are a mechanism for encapsulating and intercepting
+      A mechanism for encapsulating and intercepting
       a JavaScript application's asynchronous activity.
 
       The browser DOM and JavaScript have a limited number
-      of asynchronous activities, activities such as DOM events (for example, clicks),
+      of asynchronous activities, such as DOM events (for example, clicks),
       [promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), and
       [XHR](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest)
+      <!-- CF: This link goes to a mozilla page titled "XMLHttpRequest". XHR is mentioned only twice, starting
```
Comment (juleskremer) [inline public/docs/ts/latest/glossary.jade]: yes

File: public/docs/ts/latest/guide/forms.jade [hunk 10]
```diff
@@ -437,64 +437,66 @@ figure.image-display
...
-  In this example, we hide the message when the control is valid or pristine;
-  pristine means the user hasn't changed the value since it was displayed in this form.
+  In this example, you hide the message when the control is valid or pristine;
+  "pristine" means the user hasn't changed the value since it was displayed in this form.
 
   This user experience is the developer's choice. Some folks want to see the message at all times.
-  If we ignore the `pristine` state, we would hide the message only when the value is valid.
-  If we arrive in this component with a new (blank) hero or an invalid hero,
-  we'll see the error message immediately, before we've done anything.
+  <!-- CF: How about "developers" or "users" (not sure which is appropriate) instead of "folks"? -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/forms.jade]: developers

File: public/docs/ts/latest/guide/forms.jade [hunk 11]
```diff
@@ -437,64 +437,66 @@ figure.image-display
...
+  If you ignore the `pristine` state, you would hide the message only when the value is valid.
+  If you arrive in this component with a new (blank) hero or an invalid hero,
+  you'll see the error message immediately, before you've done anything.
 
   Some folks find that behavior disconcerting.
+  <!-- CF: How about "developers" or "users" (not sure which is appropriate) instead of "folks"? -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/forms.jade]: developers

**Comment by cfranger** [comment]
@juleskremer thanks for your feedback. I've updated the "Forms" and "Glossary" pages with your feedback. I also fixed a few issues that I missed on my first pass.

**Pull request merged**